clients/da-vinci-client/src/main/java/com/linkedin/davinci/store/rocksdb/RocksDBServerConfig.java

```diff
@@ -564,6 +564,10 @@ public void setBlockBaseFormatVersion(int version) {
     this.blockBaseFormatVersion = version;
   }
 
+  public void setRocksdbOptionsCompressionType(String compressionType) {
```

Comment:
Is it part of the change?

Reply:
no, but I needed it some to benchmark, I believe its good to have this setter